metadata_writer_sqlite.c

```diff
@@ -285,20 +286,40 @@ static int md_sqlite_configure(struct md_writer_sqlite *mws,
 #endif
     }
 
+    if (!strcmp(mws->parent->output_format,"sql")) {
+        if (mws->api_version == 2) {
+            dump_events = DUMP_EVENTS_V2;
+            dump_updates = DUMP_UPDATES_V2;
+        } else {
+            dump_events = DUMP_EVENTS;
+            dump_updates = DUMP_UPDATES;
+        }
+
+        dump_gps = DUMP_GPS;
+        dump_monitor = DUMP_MONITOR;
+        dump_usage = DUMP_USAGE;
+    } else {
+        dump_events = DUMP_EVENTS_JSON;
+        dump_updates = DUMP_UPDATES_JSON;
+        dump_gps = DUMP_GPS_JSON;
+        dump_monitor = DUMP_MONITOR_JSON;
+        dump_usage = DUMP_USAGE_JSON;
```

Comment:
Just to double check since I have been gone for a week, has support for the other metadata types (as json) been added to the exporter?